components/brave_rewards/browser/test/rewards_state_browsertest.cc

```diff
@@ -143,7 +143,7 @@ class RewardsStateBrowserTest : public InProcessBrowserTest {
 
   void GetTestFile(const std::string& file_name, base::FilePath* path) const {
     base::FilePath test_path;
-    base::PathService::Get(base::DIR_SOURCE_ROOT, &test_path);
+    base::PathService::Get(base::DIR_SRC_TEST_DATA_ROOT, &test_path);
```

Comment:
in tests we should `ASSERT_TRUE` or `CHECK` on these calls

Reply:
Fixed in 5e1b08d9a71f497f40cd402fbb025a0c1fe1f2dc.